proposals/bulk-memory-operations/Overview.md

```diff
@@ -296,12 +301,20 @@ implemented as follows:
 
 ### Instruction encoding
 
+All bulk memory instructions are encoded as a 0xfc prefix byte, followed by
+another opcode, optionally followed by more immediates:
+
+```
+instr ::= ...
+        | 0xfc operation:uint8 ...
+```
+
 | Name | Opcode | Immediate | Description |
 | ---- | ---- | ---- | ---- |
-| `memory.init` | `0xfc` | `0x08` | :thinking: copy from a passive data segment to linear memory |
-| `memory.drop` | `0xfc` | `0x09` | :thinking: prevent further use of passive data segment |
-| `memory.copy` | `0xfc` | `0x0a` | :thinking: copy from one region of linear memory to another region |
-| `memory.fill` | `0xfc` | `0x0b` | :thinking: fill a region of linear memory with a given byte value |
-| `table.init` | `0xfc` | `0x0c` | :thinking: copy from a passive element segment to a table |
-| `table.drop` | `0xfc` | `0x0d` | :thinking: prevent further use of a passive element segment |
-| `table.copy` | `0xfc` | `0x0e` | :thinking: copy from one region of a table to another region |
+| `memory.init` | `0x08` | `segment:varuint32`, `memory:varuint32` | :thinking: copy from a passive data segment to linear memory |
```

Comment:
The memory index should come first, and must be `0x00`. Specifying as `varuint32` implies that it could be `0x80 0x00` or other long encodings, which wouldn't match the other memory instructions in the spec (see [memory.grow](http://webassembly.github.io/spec/core/binary/instructions.html#memory-instructions) for example)